src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -18,63 +22,99 @@ const getBannerLastClosedAt = (key: string): string | undefined => {
     return (item && JSON.parse(item).value) || undefined;
 };
 
+const DEFAULT_BANNER_TIMEOUT_MILLIS = 2000;
+
+const buildNewCmpBannerConfig = (): Banner => {
+    return {
+        id: 'cmpUi',
+        canShow: () => willShowCMP().then((result) => ({ result: !!result })),
+        // New CMP is not a react component and is shown outside of react's world
+        // so render nothing if it will show
+        show: () => null,
+        timeoutMillis: null,
+    };
+};
+
+const buildOldCmpBannerConfig = (CAPI: CAPIBrowserType): Banner => {
```

Comment:
the old cmp will be going away as soon as we can get it working properly (working on it now) – if it helps you might want to hang on till then. no worries if not, we can remove it from instead once merged if that;s better for you

Reply:
but will be removed from this file entirely (I think)

Reply:
Hey thanks @sndrs :+1: yeah I figured this would probably go away at some point. As long as you're happy with it I'd vote to keep it around in this new form until we're ready to remove it (i.e. to not hold off on merging this PR).